app/Http/Controllers/V2/Dashboard/ProjectProfileDetailsController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers\V2\Dashboard;
+
+use App\Http\Controllers\Controller;
+use App\Http\Resources\V2\Dashboard\ProjectProfileDetailsResource;
+use App\Models\V2\Forms\FormOptionListOption;
+use App\Models\V2\Projects\Project;
+use Illuminate\Http\Request;
+
+class ProjectProfileDetailsController extends Controller
+{
+    public function __invoke(Request $request): ProjectProfileDetailsResource
+    {
+        if ($request->has('uuid')) {
+            $project = Project::where('uuid', $request->input('uuid'))->first();
+            $response = (object)[
+                'name' => $project->name,
+                'descriptionObjetive' => $project->objectives,
+                'country' => $this->getCountry($project->country),
+                'countrySlug' => $project->country,
+                'organisation' => $project->organisation->type,
+                'survivalRate' => $project->survival_rate,
+                'restorationStrategy' => $project->restoration_strategy,
+                'targetLandUse' => $project->land_use_types,
+                'landTenure' => $project->land_tenure_project_area,
+            ];
+
+            return new ProjectProfileDetailsResource($response);
```

Comment:
Same comment here as on the others; please drop the resource when it's not doing any work other than mapping one array onto another with the same keys.